containers/ddev-dbserver/files/docker-entrypoint.sh

```diff
@@ -74,7 +74,9 @@ rm -f /tmp/raw_mysql_version.txt
 
 # If we have extra mariadb cnf files, copy them to where they go.
 if [ -d /mnt/ddev_config/mysql ] && [ "$(echo /mnt/ddev_config/mysql/*.cnf)" != "/mnt/ddev_config/mysql/*.cnf" ] ; then
-  echo "!includedir /mnt/ddev_config/mysql" >/etc/mysql/conf.d/ddev.cnf
+  cp /mnt/ddev_config/mysql/*.cnf /etc/mysql/conf.d
+  # Ignore errors on files such as .gitmanaged
+  chmod -f -R ugo-w /etc/mysql/conf.d/*
```

Comment:
I wonder about the problems you had earlier. .gitmanaged wouldn't have been found by this chmod...